src/app/components/article/article.component.scss

```diff
@@ -0,0 +1,101 @@
+.wrapper {
+  display: grid;
+  grid-template-areas:
+    "userAvatar userName addLikeButton"
+    "userAvatar date addLikeButton"
+    "tags tags tags"
+    "articleTitle articleTitle articleTitle"
+    "articleContent articleContent articleContent"
+    "toggleArticleContentLengthButton . .";
+  grid-template-columns: min-content 1fr minmax(30px, 70px);
+  grid-template-rows: repeat(2, 2rem) auto min-content auto 2rem;
+
+  width: 60vw;
+
+  background-color: #e0e0e0;
+}
+
+.userAvatar {
+  grid-area: userAvatar;
+  justify-self: start;
+  align-self: center;
+
+  width: 60px;
+  height: 60px;
+
+  border-radius: 50%;
+  background-color: white;
+}
+
+.userName {
+  grid-area: userName;
+  justify-self: start;
+  align-self: center;
+
+  font-weight: 500;
+  color: #3F51B5;
+}
+
+.date {
+  grid-area: date;
+  justify-self: start;
+  align-self: center;
+}
+
+.addLikeButton {
+  grid-area: addLikeButton;
+  align-self: center;
+}
+
+.icon {
+  padding-right: 2px;
+}
+
+.tags {
+  grid-area: tags;
+  align-self: center;
+}
+
+.articleTitle {
+  grid-area: articleTitle;
+  align-self: center;
+
+  margin-bottom: 0;
+}
+
+.articleContent {
+  grid-area: articleContent;
+  align-self: center;
+}
+
+.toggleArticleContentLengthButton {
+  grid-area: toggleArticleContentLengthButton;
+  justify-self: start;
+  align-self: center;
+}
+
+.fullArticleContentLength {
+  max-height: 50vh;
+  overflow: auto;
+
+  &::-webkit-scrollbar {
+    width: 7px;
+  }
+
+  &::-webkit-scrollbar-track {
+    background-color: #95BBDF;
+    border-radius: 5px;
+    padding-left: 1rem;
+  }
+
+  &::-webkit-scrollbar-thumb {
+    box-shadow: inset 0 0 6px #3F51B5;
+    border-radius: 5px;
+  }
+}
+
+.limitedArticleContentLength {
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
```

Comment:
move to general scss file, and create a mixin or special class for text-overflow with ellipsis

Reply:
Agree with Mykola - it seem like it's common styles, that can be reused later.
So it would be batter to move it to general `styles.scss` file, and rename it to some abstract, like `.nowrap` OR `.ellipsis`

Reply:
And one point about CSS selectors naming:
its, not critical now (coz of training project), but traditionally CSS naming convention is uses [kebab-case](https://medium.com/@vladimirmorulus/%D1%81%D0%BF%D1%80%D0%B0%D0%B2%D0%BA%D0%B0-%D0%BF%D0%BE-case-camelcase-kebab-case-pascalcase-snake-case-%D0%B8-%D0%B2%D1%81%D0%B5-case-ecda10463fa8) for selectors naming.

it would be batter to use this approach for the future.